packages/uui-color-picker/lib/uui-color-picker.element.ts

```diff
@@ -189,11 +189,11 @@ export class UUIColorPickerElement extends LabelMixin('label', LitElement) {
     demandCustomElement(this, 'uui-input');
     demandCustomElement(this, 'uui-button');
     demandCustomElement(this, 'uui-button-group');
-    demandCustomElement(this, 'uui-popover');
     demandCustomElement(this, 'uui-color-swatches');
     demandCustomElement(this, 'uui-color-swatch');
     demandCustomElement(this, 'uui-color-area');
     demandCustomElement(this, 'uui-color-slider');
+    demandCustomElement(this, 'uui-popover-container'); //TODO: Why is the import necessary when we have this?
```

Comment:
The import should be done in the story and not of this component.

Reply:
So this is right, but the import above should be removed. :-)  There might be a few things wrong in this element, as we have several imports of other components.

Reply:
I already resolved this, as I was looking at the code anyway.. So you can just resolve when you have read my answer.